languages/elixir/exercises/concept/try-rescue-else-after/.docs/after.md

```diff
@@ -1,52 +0,0 @@
-## Errors - Key points
-
-- Rescuing errors allows functions to return defined values when it is appropriate
-- There are 4 parts to a `try-rescue` construct:
-  - The `try` block:
-    - Where the code that may raise an error is invoked
-  - The `rescue` block:
-    - Where a pattern match on the error allows the function to return a different result, or raise another error.
-  - The `else` block:
-    - Where a pattern match on the successful result from the try-block occurs
-  - The `after` block:
-    - Where functions are invoked whether the try-block succeeds or is rescued as long as the program is running.
-    - The result of the `after` block is not returned to the calling scope.
-
-```elixir
-try do
-  :a
-rescue
-  _ -> :error
-else
-  :a -> :success
-after
-  :some_action
-end
-# => :success
-```
-
-## Dynamic dispatch
```

Comment:
Technically this part should have been its own concept, but at the time of the automated migration that Erik run, our config.json was missing this concept and thus it was skipped. I attempt to fix both issues in this PR https://github.com/exercism/v3/pull/2512